app/views/curation_concerns/base/_attribute_rows.html.erb

```diff
@@ -1,12 +1,13 @@
 <%= presenter.attribute_to_html(:creator, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:contributor, label: 'Contributors', render_as: :faceted) %>
+<%= presenter.attribute_to_html(:contributor, label: t('blacklight.search.fields.show.contributor'), render_as: :faceted) %>
```

Comment:
There's no need to pass this argument. The renderer will do the translation for yoi